src/main/scala/com/webtrends/harness/component/akkahttp/logging/AccessLog.scala

```diff
@@ -10,45 +10,44 @@ trait AccessLog  {
   this: BaseCommand =>
 
   val accessLog = Logger("AccessLog")
-  def logAccess(bean: CommandBean, statusCode: Option[StatusCode] = None) {
 
-    def reconstructPath(bean: CommandBean): String = {
-      val path: String = bean.getValue[String](AkkaHttpBase.Path).getOrElse("-")
-      val segments = path.split("/").filter(_.startsWith("$"))
-      segments.foldLeft(path){(p,s) => bean.getValue[String](s.drop(1)).map(value => p.replace(s, value)).getOrElse(p)}
-    }
+  def getClientIP(bean: CommandBean): Option[String] ={
+    bean.getValue[Map[String,String]](AkkaHttpBase.RequestHeaders).flatMap (m =>
+    m.get("X_FORWARDED_FOR").orElse(m.get("REMOTE_ADDR")))
```

Comment:
I don't think this is right

1. We convert the headers to the string map you're using here, but we lowercase everything, not upper case.
2. If you want to manually check the headers, X_FORWARDED_FOR should be x-forwarded-for and needs to be able to parse out a list of addresses
3. Are we sure remote_addr will be supported? 

I would vote for just pulling the value of the "remote-address" header with a note that it requires it to be enabled in the config. It also gives a little more control for situations where someone doesn't want it logged.
`
# Enables/disables the addition of a `Remote-Address` header
# holding the clients (remote) IP address.
remote-address-header = off
`